neural_compressor/torch/utils/utility.py

```diff
@@ -15,6 +15,7 @@
 
 from typing import Callable, Dict, List, Tuple, Union
 
+import prettytable as pt
```

Comment:
I recommend using "ptable" or else, "pt" is usually used to name PyTorch related packages.

Reply:
removed https://github.com/intel/neural-compressor/pull/1876/commits/f099c927296ce705202bad3b3a37bff5893a169f
from prettytable import PrettyTable